<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="Characte_box">
      <property name="title">
       <string>Character attribute</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="man1_display">
         <property name="title">
          <string>Man1</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QProgressBar" name="man1_health">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QProgressBar::chunk {
     background-color: #FF0000;
	alignment: middle;
 }</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="man1_mana">
            <property name="styleSheet">
             <string notr="true">QProgressBar::chunk {
     background-color: #0000FF;
 }</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="man2_display">
         <property name="title">
          <string>Man2</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QProgressBar" name="man2_health">
            <property name="styleSheet">
             <string notr="true">QProgressBar::chunk {
     background-color: #FF0000;
 }</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="man2_mana">
            <property name="styleSheet">
             <string notr="true">QProgressBar::chunk {
     background-color: #0000FF;
 }</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="man3_dispaly">
         <property name="title">
          <string>Man3</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QProgressBar" name="man3_health">
            <property name="styleSheet">
             <string notr="true">QProgressBar::chunk {
     background-color: #FF0000;
 }</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="man3_mana">
            <property name="styleSheet">
             <string notr="true">QProgressBar::chunk {
     background-color: #0000FF;
 }</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="man4_display">
         <property name="title">
          <string>Man4</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QProgressBar" name="man4_health">
            <property name="styleSheet">
             <string notr="true">QProgressBar::chunk {
     background-color: #FF0000;
 }</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="man4_mana">
            <property name="styleSheet">
             <string notr="true">QProgressBar::chunk {
     background-color: #0000FF;
 }</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
